lib/engine/step/base.rb

```diff
@@ -126,6 +126,10 @@ def try_take_player_loan(entity, price); end
       def inspect
         "<#{self.class.name}>"
       end
+
+      def game_buttons(_entity)
```

Comment:
Use step.respond_to?(:game_buttons) instead of adding to base.

Reply:
Looks like you already do in the view, so I believe the change to this file isn't needed.

Reply:
I probably waffled back and forth and did some of each way.  Oops.